src/components/FiltersDrawer/FiltersDrawer.tsx

```diff
@@ -33,9 +38,9 @@ export type Props = {
    */
   onApply?: () => void;
   /**
-   * Sets the filters drawer window open or closed.
+   * Text to be placed in the button that activates the Filters Drawer
    */
-  isActive: boolean;
+  triggerText?: string;
```

Comment:
I know this api was just moved over from `Filters` but, looking at this PR, I'm wondering about how much control consumers have over the look and feel of this trigger button. That seems like the kind of thing that gets shifted around in designs because designers are used to having a lot of control over buttons. In the PR description you said

>If custom trigger buttons are needed, scrums can just use regular Button and follow similar implementation

but I'm just wondering how much work that would take / what that would look like. Maybe we could have a story with that example.

Reply:
Hmmm I think this is a valid point. I'll make the `triggerText` a required `trigger: ReactNode` instead and have a `FiltersButton` subcomponent that can be used as the trigger and update the stories to reflect it. Also will add another story with another CTA kind of element to show other components can be used

Reply:
Awesome, thanks! Feel free to punt that to a separate PR though.

Reply:
[Ticket](https://app.shortcut.com/czi-edu/story/215710/filtersbutton-subcomponent-and-change-trigger-to-default-to-filtersbutton) created so I don't lose track